easybuild/easyblocks/n/ncurses.py

```diff
@@ -34,18 +36,21 @@ def configure_step(self):
     def sanity_check_step(self):
         """Custom sanity check for ncurses."""
 
+        libs = ["form", "menu", "ncurses", "panel"]
+
         custom_paths = {
-                        'files' : ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
-                                                          "reset", "tabs", "tic", "toe", "tput", "tset"]] +
-                                  ['lib/lib%s.a' % x for x in ["form", "form", "menu", "menu_g", "ncurses", "ncurses++",
-                                                               "ncurses_g", "panel", "panel_g"]],
-                        'dirs' : ['include']
-                       }
+            'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
+                                             "reset", "tabs", "tic", "toe", "tput", "tset"]] +
+                     ['lib/lib%s%s.a' % (x, y) for x in libs for y in ['', '_g', 'w', 'w_g']] +
+                     ['lib/lib%s%s.so' % (x, y) for x in libs for y in ['', 'w']] +
+                     ['lib/libncurses++%s.a' % x for x in ['', 'w']],
+            'dirs': ['include', 'include/ncursesw']
```

Comment:
so, can you tackle what needs to be done in the `ncurses` easyconfigs PR you're working on then?

i) adding `easyblock = 'ConfigureMake'` in all of them
ii) add `sanity_check_paths` in all of them
iii) adding the `CPLUS` thing in all of them

Once that is covered, we can re-evaluate dropping this easyblock like a stone, and having a proper way of doing error reporting on this one (specifically).